_posts/2019-06-04-bitcoin-and-the-store-of-value-narrative.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: "Bitcoin and the Store of Value Narrative"
+author: Manuel Polavieja
+excerpt: "Manuel posits that Bitcoin's value cannot be explained with a store-of-value approach. <br><br>"
+---
+
+<hr>
+
+_Manuel writes at juandemariana.org, is a perpetual student of monetary theory, and is a Bitcoin quasi-maximalist (he is almost there)._
```

Comment:
Would you mind making "juandemariana.org" a link to the actual website? I think I forgot to do that in my original PR.